src/pages/Ingame/index.js

```diff
@@ -111,11 +122,17 @@ export default function Game() {
 
   const handleSelect = (e) => {
     setIsSubmitted(false);
+    setIsChecked(true);
     setMycard(e.target.htmlFor);
```

Comment:
단순히 x,y의 텍스트로 판단하시는 거면 htmlFor보단 아래처럼 추천드립니다.
label의 id로 판단하시는 거면 아래와 같이 사용하셔도 될거 같아요
```suggestion
    setMycard(e.target.innerText);
```